src/components/tabs/tabs.ts

```diff
@@ -142,6 +142,8 @@ export default class IgcTabsComponent extends EventEmitterMixin<
 
     await this.updateComplete;
 
+    this.activeTab = this.tabs.find((tab) => tab.selected);
+    this.updateSelectedTab();
```

Comment:
It seems like you can just move this call after the `setSelectedTab` so it'll properly assign the visibility of the panels

Reply:
Could also use a name change it seems, since the name of the method doesn't really correlate with what it's doing (fixing up the panels)